scheds/rust/scx_lavd/src/bpf/intf.h

```diff
@@ -123,6 +123,7 @@ struct task_ctx {
 	s8	futex_boost;		/* futex boost count */
 	u8	is_greedy;		/* task's overscheduling ratio compared to its nice priority */
 	u8	need_lock_boost;	/* need to boost lock for deadline calculation */
+	u8	lock_holder_xted;	/* slice is already extended for a lock holder task */
```

Comment:
Thank you for the comment @hodgesds ! I think so far it is okay since the size of task_ctx fits into 2 cache lines. When I should add one more byte, I will change them to bitmask. The same applies to cpu_ctx. :-)